docs/visual-basic/getting-started/whats-new.md

```diff
@@ -15,45 +15,45 @@ This topic lists key feature names for each version of Visual Basic, with detail
 
 ## Current version
 
-Visual Basic 16.0 / Visual Studio 2019 Version 16.0  
+Visual Basic 16.0 / Visual Studio 2019 version 16.0
```

Comment:
The whitespace that was removed here was significant.  It was used to generate a hard new line in the rendered page.  See [current site](https://docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/whats-new) vs. [this change](https://review.docs.microsoft.com/en-us/dotnet/visual-basic/getting-started/whats-new?branch=pr-en-us-15291)

That applies to all changes in this file.

Reply:
Oopsies. Would it be okay if I add a hard line break using a \ character?

Reply:
I think so. @mairaw might have a preferred suggestion.